### PR TITLE
Javascript javascript1 week4

### PR DESCRIPTION
**Pull request by andrii1**

### CODE REVIEW

File: javascript/javascript1/week4/homework.js [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+const todos = [];
+
+function getReply(command) {
+	if (typeof command !== "string" || command === "") {
+		throw new Error("Command must be a non-empty string");
+	}
+
+	if (command === "Hello my name is Benjamin" && name === "") {
+		name = "Benjamin";
+		return "Nice to meet you Benjamin";
```
Comment (SlaviaF) [inline javascript/javascript1/week4/homework.js:12]: Great effort. However the idea is not to hard code the name. What happens when you console with a different name. Maybe  try checking if the string includes "Hello my name is" on line 10 and try working with it .

File: javascript/javascript1/week4/homework.js [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+
+	if (command === "What should I wear today?") {
+		const clothes = ["jeans", "shirt", "pants", "hat", "shorts"];
+		const random = clothes[Math.floor(Math.random() * clothes.length)];
+		return `You should wear ${random}`;
+	}
+}
+
+//getReply(command);
+console.log(getReply("Hello my name is Benjamin"));
```
Comment (SlaviaF) [inline javascript/javascript1/week4/homework.js:91]: if I console.log(getReply("Hello my name is Benjamin")); It will give me undefined.

**Review by SlaviaF — COMMENTED** [review]
Great stuff Andrii.

File: javascript/javascript1/week4/homework.js (hunk 1 above)
Comment (andrii1) [reply javascript/javascript1/week4/homework.js:12]: ok thank you!

File: javascript/javascript1/week4/homework.js (hunk 2 above)
Comment (andrii1) [reply javascript/javascript1/week4/homework.js:91]: hm

File: javascript/javascript1/week4/homework.js (hunk 1 above)
Comment (andrii1) [reply javascript/javascript1/week4/homework.js:12]: done

File: javascript/javascript1/week4/homework.js (hunk 2 above)
Comment (andrii1) [reply javascript/javascript1/week4/homework.js:91]: works in my console..

Comment (andrii1) [reply javascript/javascript1/week4/homework.js:91]: `	if (command.includes("Hello my name is") && name === "") {
		name = command.substring(command.indexOf("is") + 3);
		return name;
	} else if (command.includes("Hello my name is") && name != "") {
		return "You already added your name";
	}`

Comment (andrii1) [reply javascript/javascript1/week4/homework.js:91]: don't know how to solve it..

**Pull request merged**